Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/DiagnosticsTab/DiagnosticsTabContent.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { useRecoilValue, useSetRecoilState } from 'recoil';
+
+import { currentProjectIdState, exportSkillModalInfoState } from '../../../../../recoilModel';
+import { navigateTo } from '../../../../../utils/navigation';
+
+import { DiagnosticList } from './DiagnosticList';
+import { useDiagnosticsData } from './useDiagnostics';
+
+export const DiagnosticsContent = () => {
+  const skillId = useRecoilValue(currentProjectIdState);
```

Comment:
const projectId?

Reply:
removed skillId usage since DiagnosicsList no longer relies on it